02/mh70cz/game.py

```diff
@@ -0,0 +1,130 @@
+
+# Code Challenge 02 - Word Values Part II - a simple game
+# http://pybit.es/codechallenge02.html
+
+import itertools
+import random
+import string
+from collections import Counter
+
+from data import DICTIONARY, LETTER_SCORES, POUCH
+
+NUM_LETTERS = 7
+
+
+def draw_letters():
+    """Pick NUM_LETTERS letters randomly. Hint: use stdlib random"""
+    return random.sample(POUCH, k=NUM_LETTERS)
+
+
+def input_word(draw=draw_letters()):
+    """Ask player for a word and validate against draw.
+    Use _validation(word, draw) helper."""
+
+    v = False
+    while v != True:
```

Comment:
Usually you can just do `while v` or `while not v` in Python, not necessary to compare against `True` and `False`